drupal/values.yaml

```diff
@@ -97,6 +97,13 @@ nginx:
   noauthips:
     gke-internal: 10.0.0.0/8
 
+  # Security headers
+  #hsts_include_subdomains: " includeSubDomains;"
```

Comment:
Default values should be uncommented in value.yaml, and not in the templates using the `default` command. This guarantees a consistent schema, prevents drift between documentation and values,  and gives us one less level of override.

Reply:
Moved defaults to values.yml